flare/framework/tests/test_simple_games.py

```diff
@@ -48,9 +48,9 @@ def test_gym_games(self):
 
         games = ["MountainCar-v0", "CartPole-v0", "Pendulum-v0"]
         final_rewards_thresholds = [
-            -1.5,  ## drive to the right top in 150 steps (timeout is -2.0)
-            1.5,  ## hold the pole for at least 150 steps
-            -3.0  ## can swing the stick to the top most of the times
+            -150,  ## drive to the right top in 150 steps (timeout is -2.0)
```

Comment:
Why the changes?

Reply:
I removed the reward scaling in `Agent`

Reply:
This file doesn't use Agent. It's a standalone test file.